aspnetcore/data/ef-mvc/complex-data-model.md

```diff
@@ -17,7 +17,7 @@ uid: data/ef-mvc/complex-data-model
 
 By [Tom Dykstra](https://github.com/tdykstra) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-The Contoso University sample web application demonstrates how to create ASP.NET Core 1.1 MVC web applications using Entity Framework Core 1.1 and Visual Studio 2017. For information about the tutorial series, see [the first tutorial in the series](intro.md).
+The Contoso University sample web application demonstrates how to create ASP.NET Core MVC web applications using Entity Framework Core and Visual Studio. For information about the tutorial series, see [the first tutorial in the series](intro.md).
```

Comment:
I don't mind changing convoluted relative links to xref, but otherwise I don't think we're requiring that every otherwise untouched relative link needs to change to xref.